smartrouter/scripts/smartrouter/install

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+DISTRIBUTION_ZIP=jboss-bpmsuite-7.0.0.Alpha01-engine.zip
+DISTRIBUTION_ARTIFACT=kie-server-router-proxy-7.1.0.Beta3-redhat-1.jar
+DISTRIBUTION_ROUTER_PATH=jboss-bpmsuite-7.0-engine/${DISTRIBUTION_ARTIFACT}
+
+START_SCRIPT_NAME=start.sh
+SOURCES_DIR=/tmp/artifacts
+ROUTER_DIR=/opt/smartrouter
```

Comment:
My $0.02: I would definitely go with /opt/bpmsuite. Smartrouter is just one of many components of bpmsuite.